src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -522,9 +527,9 @@ class DeployTask extends AbstractServerTask {
 
     private void addEmbeddedLib(Element parent, Project project, LooseApplication looseApp, String dir) throws Exception {
         try {
-            if (project.configurations.getByName('compile') != null) {
+            if (project.configurations.findByName('impementation') != null) {
                 //Get only the compile dependencies that are included in the war
-                File[] filesAsDeps = project.configurations.compile.minus(project.configurations.providedCompile).getFiles().toArray()
+                File[] filesAsDeps = project.configurations.implementation.minus(project.configurations.providedCompile).getFiles().toArray()
```

Comment:
Fix the comment before this line. Also, are we sure we no longer have to look for compile dependencies (e.g. maybe when using older versions of Gradle)?

Reply:
I can add a check for both, but the compile configuration will cause an error to be thrown with Gradle 7+.